leshan-client-example/src/main/java/org/eclipse/leshan/client/example/LeshanClientExample.java

```diff
@@ -111,7 +111,7 @@ public LeshanClientExample(final String localHostName, final int localPort, fina
             public void run() {
                 if (registrationID != null) {
                     System.out.println("\tDevice: Deregistering Client '" + registrationID + "'");
-                    client.send(new DeregisterRequest(registrationID));
+                    client.send(new DeregisterRequest(registrationID), 1000);
```

Comment:
It might be worthwhile doing this too on the above RegisterRequest as that could possibly block forever if there is no server, correct?  Or at least have it printout a "Attempting Registration with no timeout".